modules/pam_access/pam_access.c

```diff
@@ -448,9 +448,11 @@ login_access (pam_handle_t *pamh, struct login_info *item)
     if ((fp = fopen(item->config_file, "r"))!=NULL) {
 	while (!match && fgets(line, sizeof(line), fp)) {
 	    lineno++;
+	    if (line[0] == 0)
+		continue;
```

Comment:
This fix goes beyond the use of size_t.

Reply:
Moved into its own PR, since this one is merely a clean up of int/size_t now: https://github.com/linux-pam/linux-pam/pull/676